DSCResources/MSFT_UserRightsAssignment/MSFT_UserRightsAssignment.psm1

```diff
@@ -171,19 +182,58 @@ function Set-TargetResource
     }
     else
     {
+        $currRights = Get-TargetResource -Policy $Policy -Identity $Identity
+
+        $Accounts = @()
+        switch ($Identity)
+        {
+            "[Local Account]" { $Accounts += (Get-CimInstance win32_useraccount -Filter "LocalAccount='True'").SID }
+            "[Local Account|Administrator]" 
+            {
+                $AdministratorsGroup = Get-CimInstance -class win32_group -filter "SID='S-1-5-32-544'"
+                $GroupUsers = get-ciminstance -query "select * from win32_groupuser where GroupComponent = `"Win32_Group.Domain='$($env:COMPUTERNAME)'`,Name='$($AdministratorsGroup.name)'`""

```

Comment:
variable names should use camel case and functions (get-ciminstance) should use pascal case.